vividus-plugin-web-app/src/main/java/org/vividus/ui/web/action/ExpectedSearchContextConditions.java

```diff
@@ -17,28 +17,56 @@
 package org.vividus.ui.web.action;
 
 import java.util.List;
+import java.util.function.Supplier;
 
 import org.openqa.selenium.By;
 import org.openqa.selenium.SearchContext;
+import org.openqa.selenium.StaleElementReferenceException;
 import org.openqa.selenium.WebElement;
 
 public final class ExpectedSearchContextConditions extends AbstractExpectedConditions<By>
 {
     @Override
     protected List<WebElement> findElements(SearchContext searchContext, By by)
     {
-        return searchContext.findElements(by);
+        return searchInforming(() -> searchContext.findElements(by));
     }
 
     @Override
     protected WebElement findElement(SearchContext searchContext, By by)
     {
-        return searchContext.findElement(by);
+        return searchInforming(() -> searchContext.findElement(by));
+    }
+
+    private <E> E searchInforming(Supplier<E> search)
+    {
+        try
+        {
+            return search.get();
+        }
+        catch (StaleElementReferenceException toWrap)
+        {
+            throw new StaleContextException(toWrap);
+        }
     }
 
     @Override
     protected String toStringParameters(By locator)
     {
         return "located by " + locator;
     }
+
+    public static class StaleContextException extends StaleElementReferenceException
+    {
+        private static final long serialVersionUID = 2449857900556530700L;
+
+        private StaleContextException(Throwable throwable)
+        {
+            super("Search context used for search is stale. "
+                + "Please double check the tests. "
+                + "You have a few options: "
+                + "1. reset context; "
+                + "2. Synchronize the tests to wait for context's stabilization;", throwable);
```

Comment:
would not line breaks make the message readable?

Reply:
![image](https://user-images.githubusercontent.com/17816729/85104735-cfe06800-b211-11ea-829b-567a3d5f6e42.png)
vs
![image](https://user-images.githubusercontent.com/17816729/85104791-ee466380-b211-11ea-8278-569f85fee7a8.png)

Looks better with linebreaks for me